docs/resources/task.md

```diff
@@ -65,6 +65,8 @@ resource "iterative_task" "example" {
 - `storage.workdir` - (Optional) Local working directory to upload and use as the `script` working directory.
 - `storage.output` - (Optional) Results directory (**relative to `workdir`**) to download (default: no download).
 - `storage.exclude` - (Optional) List of files and globs to exclude from transfering. Excluded files are neither uploaded to cloud storage nor downloaded from it. Exclusions are defined relative to `storage.workdir`.
+- `storage.container` - (Optional) Pre-allocated container to use for storage of task data, results and status.
```

Comment:
slightly wary of clashing with "docker containers" but willing to accept owing to "storage" namespace.